src/cockpit/peer.py

```diff
@@ -59,7 +59,7 @@ async def spawn(self, argv: Sequence[str], env: Sequence[str], **kwargs) -> asyn
         user_env = dict(e.split('=', 1) for e in env)
         return SubprocessTransport(loop, self, argv, env=dict(os.environ, **user_env), **kwargs)
 
-    async def start(self, init_host: Optional[str] = None) -> Dict[str, object]:
+    async def start(self, init_args: Optional[Dict[str, object]] = None) -> Dict[str, object]:
```

Comment:
I wonder if this should, itself, become a kwarg, so we could say like `.start(host='...')`.  The only thing that stops me there is the `None` possibility and how we'd choose to encode that...

Although for the status quo, the `None` possibility is marked by the absense of the `host` field, so we're certainly not removing functionality if we say "empty `kwargs`" → "no init message".